src/rebuild.go

```diff
@@ -127,13 +127,13 @@ func (a *App) Rebuild() {
     }()
   }
 
-  parse_volume := func(tvol string) {
-    for _, i := range get_files(fmt.Sprintf("http://%s/", tvol)) {
+  parse_volume := func(tvol string, userpass string) {
```

Comment:
Can you include the user pass in the volume name?

Reply:
Is this something we really want to do? On L137 we pass `tvol` into `RebuildRequest` which then gets passed into `rebuild` on L123 -> means user pass is going to get written into LevelDB?

Reply:
Same as above.